custom_components/opnsense/pyopnsense/__init__.py

```
@@ -314,6 +322,50 @@ async def is_plugin_installed(self) -> bool:
+            method, path, payload, future = await self._request_queue.get()
+            try:
+                if method == "get_from_stream":
+                    result: MutableMapping[str, Any] | list | None = await self._do_get_from_stream(
+                        path
+                    )
+                elif method == "get":
+                    result = await self._do_get(path)
+                elif method == "post":
+                    result = await self._do_post(path, payload)
+                future.set_result(result)
+            except Exception as e:  # noqa: BLE001
```

Pylint (3.3.7)

[warning] Catching too general exception Exception

(W0718)